src/allmydata/test/test_magic_folder.py

```diff
@@ -355,7 +424,8 @@ def _check_errors(n):
                             MagicFolder, client, 'bad', '', errors_dir, magicfolderdb, inotify=fake_inotify)
             self.shouldFail(AssertionError, 'non-directory upload.dircap', 'does not refer to a directory',
                             MagicFolder, client, 'URI:LIT:foo', '', errors_dir, magicfolderdb, inotify=fake_inotify)
-            self.shouldFail(AssertionError, 'readonly upload.dircap', 'is not a writecap to a directory',
+            self.shouldFail(AssertionError, 'readonly upload.dircap',
+                        '\"The URI in \'private/collective_dircap\' does not refer to a directory.\"',
```

Comment:
I think this change is wrong, the test was looking for the correct error substring. (I don't know why we're no longer producing that error.)